src/components/navbar/SealVerse.tsx

```diff
@@ -0,0 +1,10 @@
+import Dropdown from 'components/Dropdown'
+
+export default function () {
+  const options = [
+    { label: 'SealCred', selected: true },
```

Comment:
May we rid of the `selected`? Or checking it in another way, like getting the host and comparing it with the link from the option? It looks strange because we have a check if the option selected inside and also give the component `selected`

Reply:
Well, I think we can throw the `selected` logic inside the component